{
	"name": "pl_error_handling",
	"properties": {
		"activities": [
			{
				"name": "df_error_handling",
				"type": "ExecuteDataFlow",
				"dependsOn": [],
				"policy": {
					"timeout": "1.00:00:00",
					"retry": 0,
					"retryIntervalInSeconds": 30,
					"secureOutput": false,
					"secureInput": false
				},
				"userProperties": [],
				"typeProperties": {
					"dataflow": {
						"referenceName": "df_error_handling",
						"type": "DataFlowReference"
					},
					"compute": {
						"coreCount": 8,
						"computeType": "General"
					},
					"traceLevel": "Fine"
				}
			}
		],
		"annotations": [],
		"lastPublishTime": "2022-07-29T08:02:22Z"
	},
	"type": "Microsoft.DataFactory/factories/pipelines"
}